ecs-proxies-deploy.yml

```diff
@@ -0,0 +1,14 @@
+#docker_service_cpu: 256
+#docker_service_memory: 512
+
+docker_service:
+    - name: eps-coordinator
+      expose: true
+      environment:
+          - name: NODE_ENV
+            value: production
```

Comment:
What does this change other than the log level? Should it be set to internal-dev?

Reply:
Different libraries can do what they want with it. The only options are "development" and "production" though, so I think everything other than your local machine is "production".